### ISSUE TITLE
A11y_AzureDataStudio_Notebooks - Code cell_Focus Order:When typing code in code cell, control elements are not in focus order

### ISSUE THREAD

**Issue by VijayaBharathyA** [microsoft/azuredatastudio#18588]
### GitHub Tags:
#A11yMAS; #A11yTCS; #A11ySev3; #DesktopApp; #BM-AzureDataStudio-Feb2022; #Azure Data Studio; #WCAG2.4.3; #Win32; #AINotInScope; #Focus Order;

### Environment Details:
Windows Version :Windows 11 Enterprise
Application name :Azure Data studio

### Repro Steps:
1.Launch Azure Data Studio Insiders application.
2.Connect to server.
3.Right-click connection and click New Notebook
4.Click on cell and select code cell.
5.Validate that all the contents & controls in code cell & toolbar 

### Actual Result:
When typing code in code cell, control elements are not in focus order 

### Expected Result:
When typing code in code cell, control elements should be in focus order

### Attachment: 

![A11y_AzureDataStudio_Notebooks - Code cell_Focus Order](https://user-images.githubusercontent.com/100189407/156115179-7f33fba5-24e1-4aa4-8705-d53e4ee5720c.gif)

**Comment by chlafreniere** [comment]
@alanrenmsft I'm a bit confused here. Could this be a "Toggle Tab Key Moves Focus" related issue (where that mode isn't turned on)?

**Comment by alanrenmsft** [comment]
@VijayaBharathyA I assume the issue is about the focus stuck in the cell editor, if that is the case, you need to run the command from command palette.
![image](https://user-images.githubusercontent.com/13777222/156256434-2a608440-42c8-4a81-bf8d-ba0fedae2063.png)

otherwise, please add some description about the expectation and actual behaviors.

**Comment by alanrenmsft** [comment]
> @alanrenmsft I'm a bit confused here. Could this be a "Toggle Tab Key Moves Focus" related issue (where that mode isn't turned on)?

@chlafreniere yes, I agree, I have closed the issue.

**Comment by VijayaBharathyA** [comment]
@alanrenmsft  I followed the instruction that you mentioned in the comment the issue is not repro

**Attachment:** 
![A11y_AzureDataStudio_Notebooks - Code cell_Focus Order_new](https://user-images.githubusercontent.com/100189407/156306433-f64ee6c9-0d3b-4546-b6f0-cc4040a1c9ae.gif)
